docs/feature_unicode.md

```diff
@@ -97,21 +88,29 @@ You can also switch the input mode by calling `set_unicode_input_mode(x)` in you
 * __UC_WIN__: (not recommended) Windows built-in Unicode input. To enable: create registry key under `HKEY_CURRENT_USER\Control Panel\Input Method\EnableHexNumpad` of type `REG_SZ` called `EnableHexNumpad`, set its value to 1, and reboot. This method is not recommended because of reliability and compatibility issue, use WinCompose method below instead.
 * __UC_WINC__: Windows Unicode input using WinCompose. Requires [WinCompose](https://github.com/samhocevar/wincompose). Works reliably under many (all?) variations of Windows.
 
-### Keycodes
+!> There is an input mode option for BSD, but it's not currently implemented. If you use BSD and would like support for this, please [open an issue on GitHub](https://github.com/qmk/qmk_firmware/issues).
+
+### Switching Input Modes
 
-|Key                       |Aliases  |Input Method  |Description                                        |
-|--------------------------|---------|--------------|---------------------------------------------------|
-|`UNICODE_MODE_OSX`        |`UC_M_OS`|`UC_OSX`      |Sets the input method for MacOS X                  |
-|`UNICODE_MODE_LNX`        |`UC_M_LN`|`UC_LNX`      |Sets the input method for Linux                    |
-|`UNICODE_MODE_BSD`        |`UC_M_BS`|`UC_BSD`      |Sets the input method for BSD (Non-Operational)    |
```

Comment:
I would prefer to have this in there, so it's at least documented.   It may encourage somebody to help us add support for it.

Reply:
Will add now.